vars/runATH.groovy

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env groovy
+
+/**
+ * Wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def String athRevision = params.get('athRevision', 'master')
+    def String metadataFile = params.get('metadataFile', 'essentials.yml')
+    def String jenkins = params.get('jenkins', 'latest')
+
+    def mirror = "http://mirrors.jenkins.io/"
```

Comment:
Do we foresee any connectivity issues when using mirrors?

Reply:
I only expect that to happen with "latest" versions, the other ones are not going to be used too much and concrete versions are downloaded via maven. For "latest" I am using CURL and following redirects (as latest is not deterministic to be downloaded via maven) , we may have problems if the number of executions of runATH is big enough at a given instant of time to overhelm the download server, but I have not found an easy alternative, as crawling the download to get the latest version and use maven could potentially have the same problem